templates/viewmodel/templates/portfolio_project_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "viewmodel/templates/base.html" %}
+
+{% block content %}
+        <h1>Portfolio Project List</h1>
```

Comment:
whooping change indent here.  Add an issue to add a bear to standardise our .html spacing.  Possibly do this one yourself, so it is done quickly.